modules/overview-of-kueue-resources.adoc

```diff
@@ -130,9 +116,16 @@ spec:
         nominalQuota: '5'
 ----
 
+A cluster administrator should notify the consumers of a cluster queue about the quota limits for that cluster queue.
 The cluster queue starts a distributed workload only if the total required resources are within these quota limits. 
 If the sum of the requests for a resource in a distributed workload is greater than the specified quota for that resource in the cluster queue, the cluster queue does not admit the distributed workload.
 
+ifndef::upstream[]
+See the example cluster-queue configurations provided in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing_distributed_workloads#ref-example-kueue-resource-configurations_distributed-workloads[Example Kueue resource configurations].
```

Comment:
`cluster-queue` or cluster queue ?

Reply:
Removed the text entirely, it's not needed.